packages/yum.go

```diff
@@ -131,6 +134,35 @@ func parseYumUpdates(data []byte) []*PkgInfo {
 	return pkgs
 }
 
+func getYumTXFile(data []byte) string {
+	/* The last lines of a non-complete yum update where the transaction
+	   is saved look like:
+
+	     Exiting on user command
+	     Your transaction was saved, rerun it with:
+	      yum load-transaction /tmp/yum_save_tx.2022-10-12.20-26.j3auah.yumtx
+	*/
+	lines := bytes.Split(bytes.TrimSpace(data), []byte("\n"))
+
+	var filename string
+	for _, ln := range lines {
```

Comment:
I'd suggest a change, but feel free to ignore it (it's more a readability/stylistic suggestion):

```go
	for _, ln := range lines {
		flds := bytes.Fields(ln)
		if len(flds) == 3 && string(flds[1]) == "load-transaction" &&  strings.HasPrefix(filename, "/tmp/yum_save_tx.") {
			return string(flds[2])
		}
	}
```

Reply:
Updated. The original was easier to develop with, but once I had a unit test for it, I think this is a little cleaner. (I didn't copy/paste your suggestion, but it's mostly the same.)